modules/cnf-running-the-performance-creator-profile.adoc

```diff
@@ -47,14 +47,14 @@ Password: ************
 
 . Optional: Display help for the PPC tool:
 +
-[source,terminal]
+[source,terminal,subs="attributes+"]
 ----
-$ podman run --entrypoint performance-profile-creator registry.redhat.io/openshift4/ose-cluster-node-tuning-operator:v4.11 -h
+$ podman run --rm --entrypoint performance-profile-creator registry.redhat.io/openshift4/ose-cluster-node-tuning-operator:v{product-version} -h
 ----
 +
 .Example output
 +
-[source,terminal]
+[source,terminal, ]
```

Comment:
Is there supposed to be something after the comma here?

Reply:
hmm.. let me look